src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
The return value should have an optional wrapper if the method can return `null`.
 ```suggestion
     Optional<Manufacturer> get(Long id);
```